docs/guide/en/source/appendixes/optionaltasks.xml

```diff
@@ -12077,6 +12077,13 @@ host="webserver" command="ls" /></programlisting>
                         <entry><literal>false</literal></entry>
                         <entry>No</entry>
                     </row>
+                    <row>
+                        <entry><literal>configOption</literal></entry>
+                        <entry><literal role="type">String</literal></entry>
+                        <entry>Override subversion's config option</entry>
+                        <entry><literal>false</literal></entry>
```

Comment:
Please change `false` to `n/a` as it could suggest a bool type. (same for all other occurrencies)
Also a small example would be good to have.
Is there also any change to have a unit test for it?